ingeniamotion/wizard_tests/feedbacks_tests/digital_hall_test.py

```diff
@@ -17,17 +22,19 @@ class DigitalHallTest(Feedbacks):
 
     SENSOR_TYPE_FEEDBACK_TEST = SensorType.HALLS
 
-    def __init__(self, mc, servo, axis):
+    def __init__(self, mc: "MotionController", servo: str, axis: int) -> None:
         super().__init__(mc, servo, axis)
-        self.backup_registers_names += self.BACKUP_REGISTERS_HALLS
+        self.backup_registers_names.extend(self.BACKUP_REGISTERS_HALLS)
 
     @BaseTest.stoppable
-    def feedback_setting(self):
+    def feedback_setting(self) -> None:
         self.halls_extra_settings()
         super().feedback_setting()
 
     @BaseTest.stoppable
-    def halls_extra_settings(self):
+    def halls_extra_settings(self) -> None:
+        if not isinstance(self.pair_poles, int):
```

Comment:
Probably should be more accurate check  if pair_poles is not None, it is already defined as Optional[int]

Reply:
I think it's already typed well. If pair_poles variable was a None, the register couldn't be set correctly. In addition, the set_register function doesn't accept None type. Maybe the best solution would be typing pair_poles to int instead. What do you think?